package.json

```diff
@@ -45,5 +45,10 @@
   },
   "optionalDependencies": {
     "js-beautify": "^1.13.11"
+  }, 
+  "overrides": {
```

Comment:
This forces a package bump of `cross-spawn` from vulnerable version 7.0.3 to patched version 7.0.5, when/if express-hb gets an actual patch, this can be removed when express-hbs is updated in this repo.